winston.js

```diff
@@ -1,11 +1,15 @@
-const winston = require("winston");
+const { createLogger, format, transports } = require("winston");
+const { combine, timestamp, prettyPrint } = format;
+
 const config = require("./config");
+
 const loggingLevel = config.logLevel;
 
-const logger = winston.createLogger({
+const logger = createLogger({
+  label: timestamp(),
   level: loggingLevel,
-  format: winston.format.json(),
-  transports: [new winston.transports.Console()],
+  format: combine(timestamp(), prettyPrint()),
```

Comment:
This is okay, but I do think we should come back to it asap. I think our goal here should be non-human log parsing